config/types/etcd.go

```diff
@@ -126,13 +128,16 @@ func init() {
 					Contents: contents,
 				}},
 			})
+			if hasTemplating && platform == templating.PlatformOpenStackMetadata {
```

Comment:
I don't like that we have to remember to include this everywhere templating is used. Is there anyway to abstract this so that it's handled in one place. That would also relieve the need for that hideous global state variable.